commons-packet/commons-packet-manager/src/test/java/io/mosip/commons/packet/test/dto/packet/RegistrationPacketTest.java

```diff
@@ -0,0 +1,82 @@
+package io.mosip.commons.packet.test.dto.packet;
+
+import io.mosip.commons.packet.dto.packet.RegistrationPacket;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+
+public class RegistrationPacketTest {
+    @Autowired
+    private RegistrationPacket registrationPacket;
+    @Before
+    public void befor(){
+        registrationPacket =new RegistrationPacket("yyyy-MM-dd'T'HH:mm:ss.SSS");
+    }
+
+    @Test
+    public void setFilesWithArrayOfStringTest()
```

Comment:
Test case naming convention should be <methodToTest>_<TestCaseName>_<TestResult>
Eg: setField_withArrayOfString_thenPass

Reply:
Kindly change the method names of all the test cases defined in this RegistrationPacketTest

Reply:
done